common/src/main/java/tachyon/Constants.java

```diff
@@ -97,22 +97,23 @@
   public static final String LOGS_DIR = "tachyon.logs.dir";
   public static final String UNDERFS_ADDRESS = "tachyon.underfs.address";
   public static final String UNDERFS_HDFS_IMPL = "tachyon.underfs.hdfs.impl";
-  public static final String UNDERFS_HADOOP_CONFIGURATION = "tachyon.underfs.hadoop.configuration";
```

Comment:
changes in this file are totally automatic. Please review `common/src/main/resources/tachyon-default.properties` which has real changes.